RealifeTech/RealifeTech.swift

```diff
@@ -44,7 +44,7 @@ public class RealifeTech {
             deviceHelper: deviceHelper,
             reachabilityHelper: reachabilityChecker,
             apiHelper: apiHelper,
-            graphQLManager: graphQLManager,
+            graphQLManager: GraphQLManager.shared,
```

Comment:
As line 39 `GraphQLFactory.makeGraphQLManager` already return singleton `GraphQLManager.shared` to local property `graphQLManager`, may I know the reason why passing `GraphQLManager.shared` instead please (also the below changes) ?
Note: probably missed `graphQLManager` in line 59

Reply:
I forgot to revert them back 😅